calc/.eslintrc

```diff
@@ -2,8 +2,9 @@
   "root": true,
   "extends": "@pkmn",
   "rules": {
+    "max-len": ["error", { "code": 100, "ignoreUrls": true }],
```

Comment:
Is this rule change worth upstreaming?

Reply:
Would readily accept it into https://github.com/pkmn/eslint-config (I didn't know ignoreUrls was an option), though in this particular PR this style change is unnecessary :)